site/app/templates/admin/Configuration.twig

```diff
@@ -304,5 +304,36 @@
                 <span class="option-alt">Show a custom message to students at the top of the navigation page.</span>
             </label>
         </div>
+
+        <br/>
+        <br/>
+        <div class="danger">
+            <h2> Danger Zone</h2>
+            <hr>
+                <div class="config-row checkbox-row">
+                    <input type="checkbox" name="all_self_registration" id="all-self-registration" data-testid="all-self-registration" value="true" {{ fields['self_registration_type'] == constant('app\\controllers\\admin\\ConfigurationController::ALL_SELF_REGISTER') ? 'checked' : '' }}
+                    onclick="return confirmSelfRegistration(this, {{ fields['registration_sections'] is empty ? 'true' : 'false' }})" />
```

Comment:
It might be more readable to break this into separate lines where each attribute is on a separate line.